docs/dev/api_usage_examples/common.py

```diff
@@ -0,0 +1,54 @@
+'''
+common.py
+'''
+
+from enum import Enum
+from time import time
+from datetime import datetime
+
+ROOT = '127.0.0.1'
+API_VERSION = 1
+TOKEN = '93fb3a42-9177-48a0-b07a-e8a7019218f6' # Replace with your dev token.
```

Comment:
Sharing a token is not a good idea, should be replaced with some boilerplate.

Reply:
I know, but still feels weird to share ANY actual token.

Reply:
Changed it to `YOUR_TOKEN_HERE`.